packages/host/tests/unit/index-query-engine-test.ts

```diff
@@ -484,15 +484,6 @@ module('Unit | query', function (hooks) {
     });
   });
 
-  test('can get paginated results that are stable during index mutations', async function (assert) {
-    await runSharedTest(indexQueryEngineTests, assert, {
-      indexQueryEngine,
-      dbAdapter,
-      loader,
-      testCards,
-    });
-  });
-
```

Comment:
when ed ad I reviewed this he felt that this was actually a weird feature in that this means that as you paginate the search results you are not necessarily seeing the latest query results since we are trying to maintain the results from when you originally made the request as you paginate thru the results as opposed to showing the latest values.